Module IA: undefined name 'listen'

./main.py:18:24: F821 undefined name 'listen'
            question = listen.one_phrase().as_text()
                       ^
./main.py:23:22: F821 undefined name 'ia'
            answer = ia.run(question)
                     ^
2     F821 undefined name 'listen'

--- a/.github/workflows/python-app.yml
+++ b/.github/workflows/python-app.yml
@@ -26,7 +26,7 @@ jobs:
     - name: Install dependencies
       run: |+
         sudo apt-get update
-        sudo apt-get install -y portaudio19-dev
+        sudo apt-get install -y portaudio19-dev libasound-dev espeak python3-tk
         python -m pip install --upgrade pip
         pip install flake8 pytest
         [ -f requirements.txt ] && pip install -r requirements.txt

--- a/README.md
+++ b/README.md
@@ -30,7 +30,7 @@ documentation/hackathon.pdf
 #### Dependences possibles
 
 ```
-sudo apt install portaudio19-dev libasound-dev espeak
+sudo apt install portaudio19-dev libasound-dev espeak python3-tk
 ```
 
 ```

--- a/main.py
+++ b/main.py
@@ -2,14 +2,11 @@
 
 import module_audio as audio
 from robot import Robot
+
 robot = Robot()
 
-# import module_microphone as microphone
-
-# TODO an import is missing for skimage
 # import module_ia as ia
-
-# from module_microphone import listen
+from module_microphone import listen
 
 
 async def chat_bot():

--- a/requirements.txt
+++ b/requirements.txt
@@ -127,3 +127,4 @@ flask-restx==1.1.0
 sqlalchemy-media==0.17.4
 SQLAlchemy==2.0.20
 Flask-SQLAlchemy==3.0.5
+PySimpleGUI==4.60.5

